docs/docs/components.mdx

```diff
@@ -137,85 +137,6 @@ word vectors in your pipeline.
   by passing in your language identifier as the `language` option.
 
 
-### HFTransformersNLP
```

Comment:
We should not just remove the docs. I would keep it the section and add a deprecation warning, similar to https://rasa.com/docs/rasa/policies#two-stage-fallback-policy.

Reply:
I moved it to [`Deprecated Components`](
https://github.com/RasaHQ/rasa/pull/7027/files/0eca21ad7e82249ae18f07e1a27ad1fa2336d11d#diff-a1167895182b95a8cde4e98af355375597a90d92e76698823e2a61ac4d5521f8R2614)

Do you think it needs more info?

Reply:
So far we just added the deprecation warning to the section of the component itself. Not sure if we should introduce a new section for it as users might be confused.

Reply:
I'm not sure I understand, sorrry.

I introduced the new "Deprecated Components" section and moved the component headings as well as the warning to this new section. I think this is also how it is done in [https://rasa.com/docs/rasa/policies#two-stage-fallback-policy](https://rasa.com/docs/rasa/policies#two-stage-fallback-policy). Would you rather I do something else?

Reply:
ah, never mind, did not saw that, let's keep it as it is